### PR TITLE
Fixes errors in README

### PR DESCRIPTION
**Pull request by KevCron**
Let me know if this is the wrong base branch to be merging into.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,20 +119,20 @@ Calls to both `fetch()` and `get()` emit one value and then call `onCompleted()`
...
+To prevent duplicate requests for the same data, Store offers an inflight debouncer. If the same request is made within a minute of a previous identical request, the same response will be returned. This is useful for situations when your app needs to make many async calls for the same data at startup or when users are obsessively pulling to refresh. As an example, The New York Times news app asynchronously calls `ConfigStore.get()` from 12 different places on startup. The first call blocks while all others wait for the data to arrive. We have seen a dramatic decrease in the app's the data usage after implementing this in flight logic.
 
 
 ### Adding a Parser
 
-Since it is rare for data to arrive from the network in the format that your views need, Stores can delegate to a parser by using a `StoreBuilder.<BarCode, BufferedSource, Article>parsedWithKey()
+Since it is rare for data to arrive from the network in the format that your views need, Stores can delegate to a parser by using a `StoreBuilder.<BarCode, BufferedSource, Article>parsedWithKey()``1
```
Comment (ramonaharrison) [inline README.md]: Is the ``1 a typo here?

Comment (tiwiz) [inline README.md]: The final `1 looks a bit off, could it be a typo?

**Review by tiwiz — CHANGES_REQUESTED** [review]

**Comment by KevCron** [comment]
Removed the typo, good catch!

**Review by tiwiz — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -119,20 +119,20 @@ Calls to both `fetch()` and `get()` emit one value and then call `onCompleted()`
...
+To prevent duplicate requests for the same data, Store offers an inflight debouncer. If the same request is made within a minute of a previous identical request, the same response will be returned. This is useful for situations when your app needs to make many async calls for the same data at startup or when users are obsessively pulling to refresh. As an example, The New York Times news app asynchronously calls `ConfigStore.get()` from 12 different places on startup. The first call blocks while all others wait for the data to arrive. We have seen a dramatic decrease in the app's the data usage after implementing this in flight logic.
 
 
 ### Adding a Parser
 
-Since it is rare for data to arrive from the network in the format that your views need, Stores can delegate to a parser by using a `StoreBuilder.<BarCode, BufferedSource, Article>parsedWithKey()
+Since it is rare for data to arrive from the network in the format that your views need, Stores can delegate to a parser by using a `StoreBuilder.<BarCode, BufferedSource, Article>parsedWithKey()``
```
Comment (ramonaharrison) [inline README.md]: hate to nitpick,  but I think there's still an extra "`" that is messing up the formatting. otherwise, LGTM!

**Comment by KevCron** [comment]
@ramonaharrison @tiwiz Can I merge this?

**Comment by digitalbuddha** [comment]
Cc @wdziemia

**Review by wdziemia — APPROVED** [review]

**Pull request merged**